src/mongocxx/collection.cpp

```diff
@@ -207,7 +207,7 @@ cursor collection::find(view_or_value filter, const options::find& options) {
     }
 
     if (options.max_time()) {
-        filter_builder << "$maxTimeMS" << options.max_time()->count();
+        filter_builder << "$maxTimeMS" << bsoncxx::types::b_int64{options.max_time()->count()};
```

Comment:
Technically, std::chrono::milliseconds could be anything larger than 45 bits.  In practice, even if it were > 64bits, I suspect no user is going to put a larger value there, so the question is what happens when the compiler is told to initialize `b_int64` from a > 64-bit int type?  Cast and warn?  Error?